docs/audits/ads-in-viewport.md

```diff
@@ -1,4 +1,4 @@
-# Avoid loading ads until they are nearly on-screen
+Lazy load ads until they are nearly on-screen
```

Comment:
This seems a little off. Maybe "Lazy load ads that are off-screen"? Or "below the fold" like the link you added in the other PR.